platform/core.multitabs/src/org/netbeans/core/multitabs/impl/DocumentSwitcherTable.java

```diff
@@ -225,21 +225,27 @@ public TabData getTabData() {
         }
 
         @Override
-        public int compareTo( Object o ) {
-            if( o instanceof Item ) {
-                ProjectProxy otherProject = ((Item)o).project;
-                if( null == project && null != otherProject )
+        public int compareTo(Object o) {
```

Comment:
I think you should avoid all whitespace changes.
They make the review (here on GitHub) harder because it is difficult to find the real changes.

Reply:
I reverted formatting of method.
Thanks for review.